challenges/desafio13.js

```diff
@@ -0,0 +1,30 @@
+db.trips.aggregate([
+  {
+    $addFields: {
+      duraçao: { $subtract: ["$stopTime", "$startTime"] },
+    },
+  },
+  {
+    $match: {
+      startTime: {
+        $gte: ISODate("2016-03-10T00:00:00.000Z"),
+        $lte: ISODate("2016-03-10T23:59:59.000Z"),
+      },
+    },
+  },
+
+  {
+    $group: {
+      _id: null,
+
+      avg: { $avg: "$duraçao" },
```

Comment:
Boa!! Aqui também daria pra fazer a operação de $subtract, aí não precisaria do $addFiels lá em cima, tipo assim:
```suggestion
      avg: { $avg: { $subtract: ["$stopTime", "$startTime"] } },
```